frontend/src/components/BestExpertiseSection/Card/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button } from '../../shared/button/index';
+
+const Card = ({ title, details, button }) => {
```

Comment:
the button at the API should be an object that has the ( `text` and `url` ) key and values because it's should be clickable